about.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8"/>
+  <meta name="viewport" content="width=device-width, initial-1">
+  <meta name="description" content="IJS Summit 2022 about page">
+  <meta name="Author" content="Hemerson Foreste">
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"/>
+  <title>CC Global Summit 2015 about page</title>
+</head>
+<body> 
+    <main class="main-about">   
+      <header>
+        <div class="top-header">
+            <ul class="top-header-links">
+              <li><a class="facebook" href="" title="facebook"><i class="fab fa-facebook-f"></i></a></li>
+              <li><a class="twitter" href="" title="twitter"><i class="fab fa-twitter"></i></a></li>
+              <li>English</li>
+              <li>My Page</li>
+              <li>Logout</li>
+            </ul>
+        </div>        
+        <div class="btn-hamburger-menu mobile-version"><i class="fas fa-bars"></i></div>        
+        <div class="btn-close mobile-version"><i class="fas fa-times"></i></div> 
+        <nav class="default-navbar">  
+          <div class="logo-container">
+            <a href="index.html" class="home logo" title="home"><img src="images/LogoHemer.png" alt="logo"></a>
+          </div>                  
+          <ul class="menu">
+            <li><a href="index.html" class="home-about" title="home"><i class="fas fa-home"></i></a></li> 
+            <li><a href="about.html" class="about-about menu-items">About</a></li>
+            <li><a href="" class="program menu-items">Program</a></li>
+            <li><a href="" class="join menu-items">Join</a></li>
+            <li><a href="" class="sponsor menu-items">Sponsor</a></li>
+            <li><a href="" class="news menu-items">News</a></li>
+          </ul>
+          <button class="campaign" type="button">CC Campaign</button>
+        </nav>
+      </header>
+    
+      <section id="About-Introduction">
+        <p class="greetings">"Hello! Developers!"</p>
+        <h1 class="event-title">INTERNATIONAL JAVASCRIPT SUMMIT 2022</h1>
+        <div class="separator"></div>
+        <p class="event-description">The International JavaScript Summit brings together each 2 years the community of experts and academics.
+             From Angular and React to WebAssembly; from Progressive Web Apps to JAMstack, international experts share insights on the present and future
+              JavaScript.  Join us to broaden your knowledge and network with other JS enthusiasts.              
+        </p>
+
+        <div class="contact-us">
```

Comment:
- [ ] **Required** In the Mobile version, This div is going outside the block, Kindly fix it. 🔧 
- Your website
![jtyj](https://user-images.githubusercontent.com/26767383/152565984-28a2b618-4c9b-42f9-93c3-4e02c6ad7365.PNG)
 - Original website
![efwf](https://user-images.githubusercontent.com/26767383/152566281-1b9d83d4-1e94-4b43-aae3-034e33eeaf15.PNG)

Reply:
- [ ] On smaller screens, you still have texts overlapping each other on the about page. check the screenshot below
![Screenshot from 2022-02-05 04-41-15](https://user-images.githubusercontent.com/20245164/152624034-0b89c36a-f4fc-47a0-9c01-860931b969a3.png)
Kindly fix it.